src/sentry/static/sentry/app/views/organizationEventsV2/modalLineGraph.jsx

```diff
@@ -93,11 +93,13 @@ const handleClick = async function(
     start: getUtcDateString(value),
     end: getUtcDateString(value + intervalToMilliseconds(interval)),
     limit: 1,
-    sort: sortField,
     referenceEvent: `${currentEvent.projectSlug}:${currentEvent.eventID}`,
     query: queryString,
     field,
   };
+  if (sortField !== null) {
```

Comment:
can `sortField` be other falsey values?

Reply:
It shouldn't There is a conditional above that pins it to `timestamp` or `last_seen`.